internal/controller/apiextensions/controller/options.go

```diff
@@ -35,10 +35,6 @@ type Options struct {
 	// private registry authentication when pulling Composition Functions.
 	ServiceAccount string
 
-	// Registry is the default registry to use when pulling containers for
-	// Composition Functions
-	Registry string
```

Comment:
i believe this was leftover from the alpha implementation of functions that had specific machinery with a function runner and rootless containers. Now it more or less relies on the package manager to download/install/run a function in a deployment.

Reply:
You're right. Looks like it was added in https://github.com/crossplane/crossplane/pull/4278.